### PR TITLE
Fix issues with accounts

### PR DESCRIPTION
**Pull request by JoshSmith**
# What's in this PR?

WIP on matching up models with API.

### CODE REVIEW

**Comment by JoshSmith** [comment]
The `bank-account` component displays a `bankAccountRoutingNumber` and `bankAccountLast4` that the API's `StripeConnectAccount` does not appear to track. This might possibly be in the `external_account`'s information, but no such relationship exists in the API.

We'll need to identify whether this is something that needs corrected on the API side.

**Comment by JoshSmith** [comment]
Appears that errors are not cleared when an identity document, for example, is uploaded with an error but then is uploaded successfully later.

File: app/controllers/project/settings/donations/payments.js [hunk 1]
```diff
@@ -107,24 +113,32 @@ export default Controller.extend({
...
+  _assignLegalEntityPersonalIdNumber(tokenData, stripeConnectAccount) {
+    set(stripeConnectAccount, 'legalEntityPersonalIdNumber', tokenData.id);
 
     return stripeConnectAccount
       .save()
       .then(RSVP.resolve)
       .catch(() => this._wrapError(ACCOUNT_UPDATE_ERROR));
   },
 
+  _creatPersonalIdNumberToken(personalIdNumber) {
```
Comment (begedin) [inline app/controllers/project/settings/donations/payments.js]: Typo here

File: app/templates/components/payments/account-setup.hbs [hunk 2]
```diff
@@ -9,9 +9,9 @@
 
 {{payments/funds-recipient
   isBusy=isBusy
+  onlegalEntityPersonalIdNumberSubmitted=(action onlegalEntityPersonalIdNumberSubmitted)
```
Comment (begedin) [inline app/templates/components/payments/account-setup.hbs]: `onlegal` -> `onLegal`, if we wanna follow the convention of other actions

Comment (JoshSmith) [inline app/templates/components/payments/account-setup.hbs]: Whoops, yep. Typo.

File: app/controllers/project/settings/donations/payments.js (hunk 1 above)
Comment (JoshSmith) [inline app/controllers/project/settings/donations/payments.js]: Hmm...how did this work?

Comment (JoshSmith) [inline app/controllers/project/settings/donations/payments.js]: Should we open an issue for an acceptance test that steps through the whole thing?

**Pull request merged**